mappings/net/minecraft/world/gen/feature/IcebergFeature.mapping

```diff
@@ -2,11 +2,21 @@ CLASS net/minecraft/class_3074 net/minecraft/world/gen/feature/IcebergFeature
 	METHOD method_13414 isAirBelow (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
 		ARG 1 world
 		ARG 2 pos
-	METHOD method_13418 (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;IIZI)V
+	METHOD method_13415 carve (IILnet/minecraft/class_2338;Lnet/minecraft/class_1936;ZDLnet/minecraft/class_2338;II)V
+	METHOD method_13416 getEllipseC (III)I
```

Comment:
what is this `EllipseC`?

Reply:
Its a Curve 
https://en.wikipedia.org/wiki/Elliptic-curve_cryptography

Reply:
then name it `getEllipticCurve`, and what is the return value?

Reply:
I updated this to be the full name of the curve
https://github.com/FabricMC/yarn/pull/1218/commits/ebcaa5a34ab6eca293c88389a5201b9bda36bd6b